apps/explorer/app/page.tsx

```diff
@@ -35,30 +36,33 @@ export default async function HomePage() {
       })
     ),
     to(
-      siaCentral.hosts({
-        params: {
-          limit: 5,
-        },
+      explored.hostsList({
+        params: { sortBy: 'date_created', dir: 'asc', limit: 500 },
+        data: { online: true },
       })
     ),
     to(explored.hostMetrics()),
     to(explored.blockMetrics()),
   ])
 
+  const selectedTopHosts = rankHosts(topHosts)
```

Comment:
This request and transformation of host data doesn't need to and probably shouldn't re-rerun on every page request, could you cache it? I think you can do this by wrapping it with `unstable_cache` and adding a revalidate time, but definitely double check the nextjs 14 docs.